backend/consensus/worker.py

```
@@ -0,0 +1,734 @@
+    Worker class for distributed consensus processing.
+    Each worker claims transactions from the database and processes them independently.
+    This reuses the exec_transaction logic from ConsensusAlgorithm.
+    """
+
+    def __init__(
+        self,
+        get_session: Callable[[], Session],
+        msg_handler: MessageHandler,
+        consensus_service: ConsensusService,
+        validators_manager: validators.Manager,
+        worker_id: str = None,
```

Ruff (0.14.0)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)